src/MessengerCustomerChat.js

```diff
@@ -95,12 +95,12 @@ export default class MessengerCustomerChat extends Component {
 
   removeFacebookSDK() {
     const fbjssdk = document.getElementById('facebook-jssdk');
-    if (fbjssdk) {
-      document.body.removeChild(fbjssdk);
+    if (fbjssdk.parentNode) {
```

Comment:
In some case, fbjssdk can be null at the first load, I suggest to check null for fbjssdk before getting parentNode

Reply:
Hey @HLcreativelab ,
Thanks for your comment. I also got this issue, and it will be fix [here](https://github.com/Yoctol/react-messenger-customer-chat/pull/23/files#diff-1e268a73a1c27b1a4e5bc26650aaca16R7). It will be merged soon.

Reply:
Great! I'm waiting for it. Thanks!

Reply:
Hi, I turned it to PR #24

Reply:
@xxhomey19 @chentsulin Great! Hope it will be available soon in 0.6.2 version. Many thanks 💯